mne_bids/tests/test_write.py

```diff
@@ -1159,3 +1159,22 @@ def test_write_anat_pathlike():
 
     # write_anat() should return a string.
     assert isinstance(anat_dir, str)
+
+
+def test_write_does_not_alter_events_inplace():
+    """Test that writing does not modify the passed events array."""
+    data_path = testing.data_path()
+    raw_fname = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc_raw.fif')
+    events_fname = (Path(data_path) / 'MEG' / 'sample' /
+                    'sample_audvis_trunc_raw-eve.fif')
```

Comment:
why not `op.join` ?

Reply:
Whoops, leftover from copying over from another test. changed that.